modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataLayoutResourceImpl.java

```diff
@@ -443,6 +444,13 @@ private Page<DataLayout> _getDataLayouts(
 
 		if (Validator.isNull(keywords)) {
 			return Page.of(
+				HashMapBuilder.<String, Map<String, String>>put(
+					"createBatch",
+					addAction(
+						ActionKeys.VIEW, ddmStructure.getStructureId(),
```

Comment:
@locpham97 @TaiPhamLR I believe the correct `actionName` should be `DataActionKeys.ADD_DATA_DEFINITION`